### PR TITLE
WIP/Not Ready For Review - (DOCSP-19892) Configure Realm Sync section - Info Related to Query Based Sync

### PR DESCRIPTION
**Pull request by mongomoe**
### Jira

- https://jira.mongodb.org/browse/DOCSP-19892

### Staged Changes (Requires MongoDB Corp SSO)

- [PAGE_NAME](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/BRANCH_NAME/)
https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-19892-Configure-Realm-Sync-Section-QBS-Epic/sync/configure/enable-sync/

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/includes/steps-configure-flexible-sync.yaml [hunk 1]
```diff
@@ -0,0 +1,88 @@
+title: Select Development Details
+ref: select-development-details
+content: |
+  To define sync rules and enable sync for your application, navigate to to the
+  :guilabel:`Sync` configuration screen through the {+leftnav+}.
+
+  .. note:: Sync Rules Cannot Change
+     
+     You define sync rules at the same time as you enable sync. Once sync is
```
Comment (MongoCaleb) [inline source/includes/steps-configure-flexible-sync.yaml:9]: Is this true for Flexible Sync? I thought not...

File: source/includes/steps-configure-flexible-sync.yaml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+     enabled, you can't change your app's sync rules unless you terminate sync
+     and re-enable it with new rules.
+---
+title: Select Queryable Fields
+ref: select-queryable-fields
+content: |
+  You can enable sync for a single :term:`linked cluster` in your application.
```
Comment (MongoCaleb) [inline source/includes/steps-configure-flexible-sync.yaml:16]: This content isn't related to selecting queryable fields.

File: source/includes/steps-configure-flexible-sync.yaml [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  :guilabel:`Select Cluster To Sync` dropdown menu.
+
+  .. figure:: /images/define-sync-rules-select-cluster.png
+     :width: 750px
+     :alt: The cluster selection dropown menu
+---
+title: Define Read & Write Permissions
+ref: define-read-write-permissions
+content: |
+  The sync :term:`partition key` is a field in every synced document that maps
```
Comment (MongoCaleb) [inline source/includes/steps-configure-flexible-sync.yaml:27]: Flexible Sync doesn't use a partition key. Is this just copy-paste for now?

Comment (mongomoe) [reply source/includes/steps-configure-flexible-sync.yaml:27]: none of this is ready for review, going to put up another PR & close this one, you can just ignore this altogether

**Comment by mongomoe** [comment]
Closing this PR as it is outdated, please refer to https://github.com/mongodb/docs-realm/pull/1538 which covers the same ticket but is up to date.

**Pull request closed without merging**